compositor_render/examples/silly_shader.rs

```diff
@@ -0,0 +1,115 @@
+use std::{collections::HashMap, process::Stdio, sync::Arc, time::Duration};
+
+use compositor_common::{
+    frame::YuvData,
+    scene::{InputSpec, NodeId, OutputSpec, Resolution, SceneSpec, TransformNodeSpec},
+    transformation::{TransformationRegistryKey, TransformationSpec},
+    Frame,
+};
+use compositor_render::{frame_set::FrameSet, Renderer};
+
+fn get_image(path: impl AsRef<std::path::Path>) -> Frame {
+    std::process::Command::new("ffmpeg")
+        .arg("-i")
+        .arg(path.as_ref().as_os_str())
+        .arg("input.yuv")
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .spawn()
+        .unwrap()
+        .wait()
+        .unwrap();
```

Comment:
It's better to use some rust lib for this, but if this is only a temporary example, you don't have to do that.

Reply:
I figured that since we're already using ffmpeg in other examples it's simpler to do this than to include a dev dependency